command-handler/src/util/get-user-data.js

```diff
@@ -8,16 +8,11 @@ export default function configUserData(serverName) {
         #cloud-config
         hostname: ${serverName}
         manage_etc_hosts: true
-        disable_root: false
```

Comment:
Did you test this? If so, awesome. I had read this was required so just want to confirm you already validated this cloud init works.